src/tools/CMakeLists.txt

```diff
@@ -16,7 +16,6 @@ else()
 endif()
 install(TARGETS rados DESTINATION bin)
 
-if(WITH_BOOST_CONTEXT AND NOT WIN32)
```

Comment:
any idea why the WIN32 part was there?

Reply:
Some people 'stubbed' it doing a windows port, but I don't know why. I put that back.